pulumi/python/kubernetes/applications/sirius/__main__.py

```diff
@@ -115,44 +115,49 @@ def add_namespace(obj):
 # We use the kubernetes namespace for this
 config = pulumi.Config('kubernetes')
 infra_type = config.require('infra_type')
-
-if infra_type == 'AWS':
-    # Logic to extract the FQDN of the load balancer for Ingress
-    ingress_project_name = pulumi_ingress_project_name()
-    ingress_stack_ref_id = f"{pulumi_user}/{ingress_project_name}/{stack_name}"
-    ingress_stack_ref = pulumi.StackReference(ingress_stack_ref_id)
-    lb_ingress_hostname = ingress_stack_ref.get_output('lb_ingress_hostname')
-    sirius_host = lb_ingress_hostname
-elif infra_type == 'kubeconfig':
-    # Logic to extract the FQDN of the load balancer for Ingress
-    ingress_project_name = pulumi_repo_ingress_project_name()
-    ingress_stack_ref_id = f"{pulumi_user}/{ingress_project_name}/{stack_name}"
-    ingress_stack_ref = pulumi.StackReference(ingress_stack_ref_id)
-    lb_ingress_hostname = ingress_stack_ref.get_output('lb_ingress_hostname')
-    # Set back to kubernetes
-    config = pulumi.Config('kubernetes')
-    lb_ingress_ip = ingress_stack_ref.get_output('lb_ingress_ip')
-    sirius_host = lb_ingress_hostname
-elif infra_type == 'DO':
-    # Logic to extract the FQDN of the load balancer for Ingress
-    ingress_project_name = pulumi_repo_ingress_project_name()
-    ingress_stack_ref_id = f"{pulumi_user}/{ingress_project_name}/{stack_name}"
-    ingress_stack_ref = pulumi.StackReference(ingress_stack_ref_id)
-    lb_ingress_hostname = ingress_stack_ref.get_output('lb_ingress_hostname')
-    # Set back to kubernetes
-    config = pulumi.Config('kubernetes')
-    lb_ingress_ip = ingress_stack_ref.get_output('lb_ingress_ip')
-    sirius_host = lb_ingress_hostname
-elif infra_type == 'LKE':
-    # Logic to extract the FQDN of the load balancer for Ingress
-    ingress_project_name = pulumi_repo_ingress_project_name()
-    ingress_stack_ref_id = f"{pulumi_user}/{ingress_project_name}/{stack_name}"
-    ingress_stack_ref = pulumi.StackReference(ingress_stack_ref_id)
-    lb_ingress_hostname = ingress_stack_ref.get_output('lb_ingress_hostname')
-    # Set back to kubernetes
-    config = pulumi.Config('kubernetes')
-    lb_ingress_ip = ingress_stack_ref.get_output('lb_ingress_ip')
-    sirius_host = lb_ingress_hostname
+ingress_project_name = pulumi_ingress_project_name()
+ingress_stack_ref_id = f"{pulumi_user}/{ingress_project_name}/{stack_name}"
+ingress_stack_ref = pulumi.StackReference(ingress_stack_ref_id)
+lb_ingress_hostname = ingress_stack_ref.get_output('lb_ingress_hostname')
+sirius_host = lb_ingress_hostname
+
+# if infra_type == 'AWS':
```

Comment:
Let's just delete the code rather than commenting it out.

Reply:
Mea Culpa; done.